.gitlab-ci.yml

```diff
@@ -72,7 +72,7 @@ test-redist:
     tags:
         - ddlog-ci-1
     script:
-        - STACK_CARGO_FLAGS='--release' stack --no-terminal test --ta "-p redist"
+        - STACK_CARGO_FLAGS='--release' stack test --ta '-p "$(NF) == \"generate redist\" || ($(NF-1) == \"compiler tests\" && $(NF) == \"redist\")"'
```

Comment:
where is this NF coming from?
Maybe a comment explaining?

Reply:
done